examples/todomvc/components/pageview.go

```diff
@@ -15,37 +15,30 @@ import (
 )
 
 type PageView struct {
-	vecty.Composite
+	*vecty.Core
 
 	Items        []*model.Item
 	newItemTitle string
 }
 
-// Apply implements the vecty.Markup interface.
-func (p *PageView) Apply(element *vecty.Element) {
-	element.AddChild(p)
-}
-
-func (p *PageView) Reconcile(oldComp vecty.Component) {
-	if oldComp, ok := oldComp.(*PageView); ok {
-		p.Body = oldComp.Body
-		p.newItemTitle = oldComp.newItemTitle
+func (p *PageView) Reconcile(prev vecty.Component) {
+	if v, ok := prev.(*PageView); ok {
+		v.newItemTitle = v.newItemTitle
 	}
-	p.RenderFunc = p.render
-	p.ReconcileBody()
+	p.Core.Reconcile(prev)
```

Comment:
We're loosing explicitness here. For example it is not possible any more to easily include logic to not call `ReconcileBody` if a re-rendering is deemed unnecessary. Adding this feature would require additional API instead of a simple `if` statement previously.